### PR TITLE
feat/318/add transaction details tests

### LINKED ISSUE #318: Add tests for TransactionDetailsCard

### PR DESCRIPTION
**Pull request by valfaro123**
### 🔥 Summary
- Add tests to TransactionDetailsCard for #318 

### 🗒️ Additional Notes
- Also updated the hash color in theme

### CODE REVIEW

File: app/src/components/cards/TransactionDetailsCard/TransactionDetailsCard.test.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    const skeletonLoaders = screen.getAllByTestId('skeleton-loader');
+
+    expect(skeletonLoaders.length).toEqual(6);
+  });
+
+  it('should render deploy details ', () => {
+    render(<TransactionDetailsCard deploy={mockDeploy} isLoading={false} />);
+
+    const amount = screen.getByTestId('deploy-amount').firstChild?.textContent;
```
Comment (mgibson-casper) [inline app/src/components/cards/TransactionDetailsCard/TransactionDetailsCard.test.tsx]: We should avoid doing any DOM traversing if possible. I would say let's either target the container/wrapper itself, and then use a `.toHaveTextContent` assertion down below. Or, target the exact element you want without traversing the DOM.

File: app/src/components/cards/TransactionDetailsCard/TransactionDetailsCard.test.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      ?.textContent;
+    const cost = screen.getByTestId('deploy-cost').firstChild?.textContent;
+    const readableTimeStamp = screen.getByTestId('readable-time-stamp')
+      .firstChild?.textContent;
+    const status = screen.getByTestId('status').firstChild?.textContent;
+    const action = screen.getByTestId('action').firstChild?.textContent;
+    const deployType =
+      screen.getByTestId('deploy-type').firstChild?.textContent;
+
+    expect(amount).toEqual('100 motes');
```
Comment (mgibson-casper) [inline app/src/components/cards/TransactionDetailsCard/TransactionDetailsCard.test.tsx]: As stated in https://github.com/casper-network/casper-blockexplorer-frontend/pull/322/files#r1196630658, we should be accessing the mock object for assertions.

**Review by Thistle-Strum — APPROVED** [review]

File: app/src/test-utils/i18n-test-init.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import en from '../../public/locales/en/translation.json';
+
+const resources = {
+  en: { translation: en },
+};
+
+i18n.use(initReactI18next).init({
+  lng: 'en',
+  fallbackLng: 'en',
+  // debug: true,
```
Comment (mgibson-casper) [inline app/src/test-utils/i18n-test-init.ts]: Do we want to leave this as a comment?

**Review by mgibson-casper — APPROVED** [review]

File: app/src/test-utils/i18n-test-init.ts (hunk 3 above)
Comment (DorianKwan) [reply app/src/test-utils/i18n-test-init.ts]: Yup -- noticed it too -- been removed

**Pull request merged**